main.py

```diff
@@ -58,6 +58,9 @@ def clean_stop(tray_icon=None):
         # Turn screen off before stopping
         display.lcd.ScreenOff()
 
+        # Turn backplate LED off for supported devices
+        display.lcd.SetBackplateLedColor(led_color=(0, 0, 0))
```

Comment:
Is there a way to call this when the task/process is killed, as [shown here](https://stackoverflow.com/a/72592788)?

Reply:
Hi @amiltonjr,
Mathoudebine talks about it here: #132

Reply:
I will also implement the atexit callback from your link in #132